test/python/utest_masarclient/test_timeStamp.py

```diff
@@ -0,0 +1,120 @@
+import unittest
```

Comment:
I have concerns about this. Some of the hardware at the beamlines claims to have such precision (I have challegend this but Wayne and Hugo say it is true). We are looking into using MASAR at the beamlines, I better keep track of this